packages/react-examples/src/react/DetailsList/DetailsList.Documents.Example.tsx

```diff
@@ -114,6 +116,13 @@ export class DetailsListDocumentsExample extends React.Component<{}, IDetailsLis
         sortDescendingAriaLabel: 'Sorted Z to A',
         onColumnClick: this._onColumnClick,
         data: 'string',
+        onRender: (item: IDocument) => {
+          return (
+            <Link onClick={this._onItemInvoked} underline>
```

Comment:
I think `onItemInvoked` needs to be passed the item data to work (it says "item invoked: undefined") right now. This should work:
```suggestion
            <Link onClick={() => this._onItemInvoked(item)} underline>
```